rust/src/adapter/otel_formatter/mod.rs

```diff
@@ -128,11 +129,11 @@ impl Span {
         start_time: SystemTime,
         end_time: SystemTime,
     ) -> Span {
-        let span_id = new_span_id();
+        let span_id = new_span_id().to_hex_16();
 
         let p_id = match parent_id {
             Some(id) => id,
-            None => new_span_id(),
+            None => new_span_id().to_hex_16(),
```

Comment:
@wwkeyboard i fixed the logic in datadog but the logic is still part of otel. not sure if this was intended or not. we can fix this in a follow up if it was not intended.

Reply:
There is a follow up PR that fixes this here: https://github.com/dylibso/observe-sdk/pull/22